### DISCUSSION TITLE
Deleted galleries still (partially) listed

### DISCUSSION THREAD

**Discussion by darkred3** [Difegue/LANraragi#407] [Q&A] [answered]
I deleted a couple of galleries manually through windows explorer. They disappeared from the gallery list as they should, but at the top of the site where it says "Showing X to Y of Z ancient chinese lithographies." they are still included. I tried restarting file watcher and cleaning temporary folders, as well as restarting LRR itself, but nothing worked.

**Comment by Difegue** [comment, marked as answer]
You need to use the "Clean Database" button in Settings to get rid of those -- By default, manually deleted galleries are not removed from the DB in case the removal happened through user error.

**Comment by darkred3** [comment]
Thank you. Tried so much but somehow skipped this button heh.

**Comment by Difegue** [comment]
NP - I'll move this to Discussions/Q&A so other users can see the answer if they have a similar question 👍
